consensus/common_test.go

```diff
@@ -72,16 +72,20 @@ func NewValidatorStub(privValidator types.PrivValidator, valIndex int) *validato
 }
 
 func (vs *validatorStub) signVote(voteType types.SignedMsgType, hash []byte, header types.PartSetHeader) (*types.Vote, error) {
+	addr, err := vs.PrivValidator.GetAddress()
+	if err != nil {
+		return nil, err
```

Comment:
errors.Wrap?

Reply:
In this case we will run into a panic shortly after. But yeah, it's more consistent and the panic would give a better indication what is going on without going through the stack trace.